lisa/spine.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+# Written by Lisa Eunyoung Lee - Jun 2022
+# Revised by Lisa Eunyoung Lee - Jan 2023
+# Spinal Cord Toolbox - Version 5.7
+
```

Comment:
Add an example of command line so that people know easily how to run the script and the orders of the arguments as well

Reply:
added https://github.com/ivadomed/canproco/pull/90/files#diff-af9974ebb3e697b7c15c512634e86604593d509f409dcc4e54f0e4393f3628eeR13